.travis.yml

```diff
@@ -24,10 +24,17 @@ before_install:
   - sudo make base apt-openfoam apt-simphony apt-lammps apt-mayavi fix-pip
 install:
   - make simphony-env
-  - make -j 3 kratos lammps jyu-lb numerrin
+  - make kratos
+  - make lammps
+  - make jyu-lb
+  - make numerrin
   - source ~/simphony/bin/activate
   - source /opt/openfoam222/etc/bashrc
   - make simphony
-  - make simphony-plugins
+  - make simphony-jyu-lb
+  - make simphony-lammps
+  - make simphony-mayavi
+  - make simphony-openfoam
+  - make simphony-kratos
```

Comment:
True.  We can still install it with `make simphony-numerri` but yeah, the tests would be skipped.